src/state/messages/convo/agent.ts

```diff
@@ -511,17 +511,17 @@ export class Convo {
   private pendingFetchConvo:
     | Promise<{
         convo: ChatBskyConvoDefs.ConvoView
-        sender: AppBskyActorDefs.ProfileViewBasic | undefined
-        recipients: AppBskyActorDefs.ProfileViewBasic[]
+        sender: ChatBskyActorDefs.ProfileViewBasic | undefined
+        recipients: ChatBskyActorDefs.ProfileViewBasic[]
```

Comment:
Good example of how we were actually using the wrong types here, but it was never an issue because they were loosely equal until now (and they're essentially the same object).